docs/content/developer/tutorials/sponsored-transactions/gas-station-server.mdx

```diff
@@ -1,5 +1,9 @@
 # The Gas Station Server
 
+:::info
+This guide is for developers who want to build their own gas station. If you'd rather just use sponsored transactions, IOTA provides a [Gas Station Component](../../../operator/gas-station/gas-station.mdx) that you can deploy in your environment.
+:::
```

Comment:
Maybe having a title in this admonition makes it nicer and more clear that this is about a tool we provide? Also docusaurus recommend to have a new line between the ::: tags and the actual content ;)

```suggestion
:::info IOTA Gas Station

This guide is for developers who want to build their own gas station. If you'd rather just use sponsored transactions, IOTA provides a [Gas Station Component](../../../operator/gas-station/gas-station.mdx) that you can deploy in your environment.

:::
```